proto/serverless/proto/instrumentation/tagset/v1/tagset.proto

```diff
@@ -0,0 +1,275 @@
+syntax = "proto3";
+
+package serverless.proto.instrumentation.tagset.v1;
```

Comment:
In schema we use _"tags"_ wording, and here we've used _"tagsset"_, is a particular reason to name it differently?

I think it'll be less confusing to rely on one wording, I'd bet on _"tags"_

Reply:
Yeah I agree, I think in my head working through `TagSet` made sense more than `Tags` but looking at now I don't see the point.